bigquery/pandas-gbq-migration/requirements.txt

```diff
@@ -7,6 +7,6 @@ pandas-gbq==0.19.2; python_version > '3.6'
 # pandas-gbq==0.14.1 is the latest compatible version for Python 3.6
 pandas-gbq==0.14.1; python_version < '3.7'
 grpcio==1.56.0
-pyarrow==13.0.0; python_version > '3.6'
+pyarrow==14.0.1; python_version > '3.6'
 # pyarrow==6.0.1 is the latest compatible version for pandas-gbq 0.14.1
-pyarrow==6.0.1; python_version < '3.7'
+pyarrow==14.0.1; python_version < '3.7'
```

Comment:
```suggestion
pyarrow==6.0.1; python_version < '3.7'
```
@Linchin

Reply:
Thanks for catching it! This got reverted by the bot again 😅